packages/js/plugins/http/src/__tests__/unit/index.test.ts

```diff
@@ -158,18 +169,21 @@ describe("test http plugin", () => {
         },
       } as AxiosResponse);
 
-      const response = await plugin.post({
-        url: "/api/test",
-        request: {
-          headers: [
-            { key: "Accept", value: "application/json" },
-            { key: "X-Test-Header", value: "test-header-value" },
-          ],
-          urlParams: [{ key: "q", value: "test-param" }],
-          body: "{request: 1001}",
-          responseType: "BINARY",
-        }
-      }, {} as Client);
+      const response = await plugin.post(
+        {
+          url: "/api/test",
+          request: {
+            headers: [
+              { key: "Accept", value: "application/json" },
```

Comment:
Why not make the headers a `Map` as well?

Reply:
Yup!